Explorer/Assets/DCL/Multiplayer/SDK/Systems/PlayerComponentsHandlerSystem.cs

```diff
@@ -67,62 +71,121 @@ private void AddPlayerIdentityData(in Entity entity, ref Profile profile, ref Ch
             {
                 Entity sceneWorldEntity = sceneEcsExecutor.World.Create();
                 var crdtEntityComponent = new CRDTEntity(crdtEntityId);
+                Avatar avatarData = profile.Avatar;
 
-                var playerIdentityData = new PlayerIdentityDataComponent
+                // TODO: Optimize with a 'PlayerSDKDataComponent' pool??
+                var playerSDKDataComponent = new PlayerSDKDataComponent
                 {
+                    IsDirty = true,
                     SceneFacade = sceneFacade,
                     SceneWorldEntity = sceneWorldEntity,
                     CRDTEntity = crdtEntityComponent,
                     Address = profile.UserId,
                     IsGuest = !profile.HasConnectedWeb3,
+                    Name = profile.Name,
+                    BodyShapeURN = avatarData.BodyShape,
+                    SkinColor = avatarData.SkinColor,
+                    EyesColor = avatarData.EyesColor,
+                    HairColor = avatarData.HairColor,
+                    WearableUrns = profile.Avatar.Wearables,
```

Comment:
Assigning the list by reference, especially cross-domain (the life-cycle of `PlayerSDKDataComponent` and `Profile` are different) can/will lead to untrackable bugs if the original list is recycled/re-used.

Reply:
Uhmm true, then just in case I updated the implementation to make a copy of those lists instead of propagating the reference: https://github.com/decentraland/unity-explorer/pull/663/commits/e5cb3931aa38ee818be2a45296f41533f4df05c7